packages/shared/src/components/squads/SquadHeaderMenu.tsx

```diff
@@ -40,6 +41,7 @@ interface SquadHeaderMenuProps {
 export default function SquadHeaderMenu({
   squad,
 }: SquadHeaderMenuProps): ReactElement {
+  const { isLoggedIn } = useContext(AuthContext);
```

Comment:
I think we also have `useAuthContext` as a hook 😬 
Never knew which one was the norm to use

Reply:
Hehe, I see you suggested that change in #2767 😂 
![image](https://github.com/dailydotdev/apps/assets/1681525/79c4edf4-459b-4f82-afec-cffc08109cc0)

Reply:
Ah I don't even look at these most of the time, just whatever autocompleted.
Probably should use the named hooks it's easier.

Let me make a suggestion here 😂